chart/templates/_helpers.yaml

```diff
@@ -997,3 +997,27 @@ capabilities:
             app: keda-operator
   {{- end }}
 {{- end }}
+
+{{- define "kubernetesVersion" -}}
+{{ printf "%s" .Capabilities.KubeVersion.Version | trimPrefix "v" }}
+{{- end -}}
+
+{{- define "airflowKerberosContainerArgs" -}}
+{{- $kubectl_version := include "kubernetesVersion" . -}}
+{{- if (semverCompare "<1.28" $kubectl_version) }}
+- airflow
+- kerberos
+- "&"
+- while [ ! -f /opt/kerberos-exit/EXIT ]; do
```

Comment:
Maybe I am missing something - but when/where the EXIT file gets created

Reply:
Just realised that I didn't push the dockerfile. But i realised there is a better way of doing it just using YAML. Made the changes, pushing it